docs/source/installation/yatai_deployment.rst

```diff
@@ -443,7 +443,7 @@ You need to configure your DNS in one of the following two options:
         .. code:: bash
 
           # Replace yatai.example.com with your domain suffix
-          kubectl -n yatai-deployment patch cm/network --type merge --patch '{"data":{"domain-suffix":"yatai.example.com"}}'
+          kubectl -n yatai-deployment patch cm/network --type merge --patch '{"data":{"domain-suffix":"'${DOMAIN_PREFIX}'"}}'
```

Comment:
`$DOMAIN_PREFIX` This environment variable does not appear

Reply:
This is already present in the existing document:

```
export DOMAIN_SUFFIX=yatai.example.com
```